charts/sophora-schema-docs/templates/deployment.yaml

```diff
@@ -32,7 +32,7 @@ spec:
             {{end}}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           volumeMounts:
-            - mountPath: /config
+            - mountPath: /data
```

Comment:
Wenn ich dich richtig verstehe, muss ja ab 5 dann die Konfig wieder in /config liege. Da könntest du hier vielleicht eine Abfrage gegen das Tag reinmachen. [Hier](https://github.com/subshell/helm-charts/blob/main/charts/sophora-schema-docs/templates/deployment.yaml#L26C8-L26C8) wird ja das Tag bestimmt mit `{{ .Values.image.tag | default .Chart.AppVersion }}`. Du könntest also im Template schauen, ob das mit "4." beginnt und wenn ja, wird /data genommen, ansonsten /config.

Reply:
Abfrage ist eingebaut. Tatsächlich handelt sich sich nur um die Version 4.0.0, in welcher der Pfad ein anderer ist. In neueren Versionen von 4. sollte der Pfad wieder ein anderer sein.

Reply:
Das passt leider noch nicht ganz. Theoretisch kann ja jemand .Values.image.tag leer lassen, dann wir die AppVersion aus der Chart.yaml genommen, was auch 4.0.0 ist. Deswegen mach das eq lieber komplett für `.Values.image.tag | default .Chart.AppVersion`.

Reply:
Hab ich auch noch mal schnell angepasst!